### PR TITLE
C84J-16: Change lon and lat to Double

### PR DESCRIPTION
**Pull request by dlazarenko-macrometa**
Changed the longitute and latitude fields in response of GET `/datacenter/local` from `String` to `Double`.

### CODE REVIEW

File: src/main/java/com/c8db/entity/DcInfoEntity.java [hunk 1]
```diff
@@ -198,22 +171,22 @@ public String getCountryName() {
         /**
          * @return the latitude
          */
-        public String getLatitude() {
+        public Double getLatitude() {
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/entity/DcInfoEntity.java:174]: @dlazarenko-macrometa - Why cannot we use Lombok getters?

File: src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java [hunk 2]
```diff
@@ -216,4 +217,22 @@ public MinReplicationFactor deserialize(final VPackSlice parent, final VPackSlic
...
+    public static final VPackDeserializer<DcInfoEntity.LocationInfo> LOCATION_INFO = new VPackDeserializer<DcInfoEntity.LocationInfo>() {
+        @Override
+        public DcInfoEntity.LocationInfo deserialize(final VPackSlice parent, final VPackSlice vpack,
+                                                     final VPackDeserializationContext context) throws VPackException {
+            String lat = vpack.get("latitude").getAsString();
+            String lon = vpack.get("latitude").getAsString();
+            final DcInfoEntity.LocationInfo locationInfo = new DcInfoEntity.LocationInfo(
+                    vpack.get("city").getAsString(),
+                    vpack.get("countrycode").getAsString(),
+                    vpack.get("countryname").getAsString(),
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java:230]: @dlazarenko-macrometa - In the `LocationInfo` class, we have `countryCode` and `countryName` in camel case. Here we have all simple case. Would this be an issue?

File: src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java [hunk 3]
```diff
@@ -216,4 +217,22 @@ public MinReplicationFactor deserialize(final VPackSlice parent, final VPackSlic
             return minReplicationFactor;
         }
     };
+
+    public static final VPackDeserializer<DcInfoEntity.LocationInfo> LOCATION_INFO = new VPackDeserializer<DcInfoEntity.LocationInfo>() {
+        @Override
+        public DcInfoEntity.LocationInfo deserialize(final VPackSlice parent, final VPackSlice vpack,
+                                                     final VPackDeserializationContext context) throws VPackException {
+            String lat = vpack.get("latitude").getAsString();
+            String lon = vpack.get("latitude").getAsString();
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java:226]:
```suggestion
            String lon = vpack.get("longitude").getAsString();
```

**Review by yehan-macrometa — CHANGES_REQUESTED** [review]

File: src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java (hunk 2 above)
Comment (dlazarenko-macrometa) [reply src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java:230]: @yehan-macrometa  - These parameters are in lowercase in API. So, there should be no issues.

File: src/main/java/com/c8db/entity/DcInfoEntity.java (hunk 1 above)
Comment (dlazarenko-macrometa) [reply src/main/java/com/c8db/entity/DcInfoEntity.java:174]: Changed

File: src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java (hunk 3 above)
Comment (dlazarenko-macrometa) [reply src/main/java/com/c8db/internal/velocypack/VPackDeserializers.java:226]: Changed

**Review by StoyanTc — APPROVED** [review]

**Review by yehan-macrometa — APPROVED** [review]

**Pull request merged**